liquibase-service/src/main/resources/db/rdb_modern/tables/022-create_nrt_interview_note-001.sql

```diff
@@ -10,6 +10,7 @@ CREATE TABLE dbo.nrt_interview_note
     user_last_name   varchar(200)                                    NULL,
     user_comment     varchar(2000)                                   NULL,
     comment_date     datetime                                        NULL,
+    record_status_cd varchar(4000)                                   NULL,
```

Comment:
As we've already deployed the last commit into 7.7.2, we might need to add a new changelog or take an approach that @alog-enquizit suggested- Adding an alter script to the interview note create script.

Reply:
Updated, could you check again and let me know what you think?

Reply:
LGTM!